docs/using/credits.md

```diff
@@ -1,39 +1,43 @@
 # Credits
 
 !!! warning "Attention"
-    On **April 1st**, ASF's On Demand Monthly Processing Quota
-    was replaced by a new **credit system**.
-    Now, you are given an allotment of **10,000 credits per month**,
-    and each type of job costs a different number of credits,
-    as shown in the table below.
-    Notably, you can now run up to 10,000 Burst InSAR jobs per month—a 10x increase!
+    On **April 1st 2024**, ASF's On Demand Monthly Processing Quota (1000 jobs per 
+    month) was replaced by a new **credit system**.
+    You are now given an allotment of **10,000 credits per month**,
+    and each type of job costs a [different number of credits to process](#credit-cost-table). 
+    Notably, you can now run up to 10,000 Burst InSAR jobs per month — 
+    a 10x increase!
 
     This change affects all ASF On Demand users,
-    whether you are accessing our service via [Vertex](/using/vertex),
-    the [HyP3 SDK](/using/sdk), or the [HyP3 API](/using/api).
+    whether you are accessing our service via [Vertex](/using/vertex){target=_blank},
+    the [HyP3 SDK](/using/sdk){target=_blank}, or the [HyP3 API](/using/api){target=_blank}.
```

Comment:
I would tend toward keeping internal `hyp3-docs` links within the same tab, but I don't think the team has a standard approach.
```suggestion
    whether you are accessing our service via [Vertex](/using/vertex),
    the [HyP3 SDK](/using/sdk), or the [HyP3 API](/using/api).
```